### ISSUE TITLE
reV CLI Version

### ISSUE THREAD

**Issue by WilliamsTravis** [NREL/reV#294]
**Bug Description**
The reV cli version command breaks, claiming to require a config. It would be nice to be able to quickly check the reV version with the cli instead of importing reV into python and checking there.

**Full Traceback**
```
(revruns)[twillia2@el3 2017]$ reV --help
Usage: reV [OPTIONS] COMMAND [ARGS]...

  reV command line interface.

Options:
  -n, --name STR          Job name. Default is "reV".
  -c, --config_file PATH  reV configuration file json for a single module.
                          [required]

  -v, --verbose           Flag to turn on debug logging. Default is not
                          verbose.

  --help                  Show this message and exit.

Commands:
  batch                     Execute multiple steps in a reV analysis...
  collect                   Collect files from a job run on multiple nodes.
  econ                      Econ analysis (lcoe, single-owner, etc...).
  generation                Generation analysis (pv, csp, windpower,...
  multi-year                Run reV multi year using the config file.
  offshore                  Offshore gen/econ aggregation with NRWAL.
  pipeline                  Execute multiple steps in a reV analysis...
  qa-qc                     Run reV QA/QC using the config file.
  rep-profiles              Run reV representative profiles using the
                            config...

  supply-curve              Run reV supply curve using the config file.
  supply-curve-aggregation  Run reV supply curve aggregation using the...
  version                   print version

(revruns)[twillia2@el3 2017]$ reV version
Usage: reV [OPTIONS] COMMAND [ARGS]...
Try 'reV --help' for help.

Error: Missing option '--config_file' / '-c'.
```

**To Reproduce**
Steps to reproduce the problem behavior
1. run `reV version`

**Expected behavior**
The version number is printed to the terminal.

**System (please complete the following information):**
 - OS: Linux
 - Compute hardware: Eagle
 - Version: 0.4.55

**Comment by MRossol** [comment]
@WilliamsTravis, this is a click issue, but I could re-organize the CLI so that the config isn't needed for reV version

**Comment by WilliamsTravis** [comment]
@MRossol Sweet, thanks.

**Comment by MRossol** [comment]
So this was stupid simple and I feel dumb for not googling this when I created the version command to start with...

**Comment by MRossol** [comment]
@WilliamsTravis , This is fixed in v0.5.

I've removed the `reV version` command now you just need to run `reV --version`
This is also true of all sub CLIs

**Comment by WilliamsTravis** [comment]
Oh great, I like that better. Thanks for hopping on this so quickly.
